src/strings/en-us.json

```
@@ -441,7 +441,9 @@
     "HeaderNewDevices": "New Devices",
     "HeaderNewRepository": "New Repository",
     "HeaderNextEpisodePlayingInValue": "Next Episode Playing in {0}",
+    "HeaderNextEpisodePlayingInValueNoAutoPlay": "Next Episode",
     "HeaderNextVideoPlayingInValue": "Next Video Playing in {0}",
+    "HeaderNextVideoPlayingInValueNoAutoPlay": "Next Video",
```

We could consolidate these a bit...
```suggestion
    "HeaderNextEpisodePlayingInValue": "Next Episode Playing in {0}",
    "HeaderNextEpisodePlayingInValueNoAutoPlay": "Next Episode",
    "HeaderNextVideoPlayingInValue": "Next Video Playing in {0}",
    "HeaderNextVideoPlayingInValueNoAutoPlay": "Next Video",
```
```suggestion
    "HeaderNextEpisodePlayingInValueNoAutoPlay": "Next Episode",
    "HeaderNextItemPlayingInValue": "{0} Playing in {1}",
    "HeaderNextVideoPlayingInValueNoAutoPlay": "Next Video",
```

Then we can pass the item type text ("Next Episode"/"Next Video") to the translate function in addition to the time.

---

There was actually no reason not to consolidate further, so I did. That header text can go back to being a ternary now with these changes instead of an if/else with nested ternaries inside of it